src/Controller/API/Schools.php

```diff
@@ -5,107 +5,16 @@
 namespace App\Controller\API;
 
 use App\Repository\SchoolRepository;
-use App\Service\ApiRequestParser;
-use App\Service\ApiResponseBuilder;
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
-use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
-use Symfony\Component\Validator\Validator\ValidatorInterface;
 
 /**
  * @package App\Controller\API
+ * @Route("/api/{version<v1|v3>}/schools")
```

Comment:
v1?

Reply:
ah crap. copy and paste error. fixing...

Reply:
fixed. :heavy_check_mark:

Reply:
that was the _only_ one i could find btw.